### ISSUE TITLE
Problem with slider in covercard

### ISSUE THREAD

**Issue by Lempomat** [mattieha/slider-button-card#213]
After the last update we cannot use the touchscreen function on the covercard on iphone and android.  We cannot slice up and down. The problem is when we configured "from top to bottom"

**Comment by stangroomd** [comment]
I concur - page scrolling seems to take precedence over the slider in the card since 2022.11. Not sure what change has caused this but it’s very sad

**Comment by Lempomat** [comment]
Yes. I found out when I made a double click and I hold the finger on the screen it works

**Comment by stangroomd** [comment]
Couldn’t get this to work reliably on android tablets (works great on iOS), ended up switching sliders to left-right

**Comment by AdmiralRaccoon** [comment]
Same issue here - Hope this gets fixed, I don´t want to slide my covers from left to right.

**Comment by sEdsKi** [comment]
Just updated to latest version. Top-bottom direction won't slide as it used to. Can push on an approx spot. Left-right direction works. I'd prefer the top bottom slider to be fixed as it makes better sense for the covers.

**Comment by mkjustuk** [comment]
Just wanted to say that I have the same issue - works ok on a browser with a mouse, but on a touch device it doesn't work.

**Comment by trapezz** [comment]
The same issue here, it worked great for more than a year that I used it on the wall tablet, but with the latest release when I updated the HA and all the add-ons it stopped working. Page scrolling takes precedence as mentioned before.

**Comment by mkjustuk** [comment]
> The same issue here, it worked great for more than a year that I used it on the wall tablet, but with the latest release when I updated the HA and all the add-ons it stopped working. Page scrolling takes precedence as mentioned before.

I moved to the later (still actively developed) branch and all was resolved.
